Sources/BraveWallet/Crypto/NFT/NFTDetailView.swift

```diff
@@ -60,129 +78,237 @@ struct NFTDetailView: View {
-            } else {
+            } label: {
               HStack {
-                Text(Strings.Wallet.tokenMintAddress)
-                  .font(.headline.weight(.semibold))
-                Spacer()
-                Button(action: {
-                  if WalletConstants.supportedTestNetworkChainIds.contains(nftDetailStore.networkInfo.chainId) {
-                    if let components = nftDetailStore.networkInfo.blockExplorerUrls.first?.separatedBy("/?cluster="), let baseURL = components.first {
-                      let cluster = components.last ?? ""
-                      if let nftURL = URL(string: "\(baseURL)/address/\(nftDetailStore.nft.contractAddress)/?cluster=\(cluster)") {
-                        openWalletURL(nftURL)
-                      }
-                    }
-                  } else {
-                    if let explorerURL = nftDetailStore.networkInfo.blockExplorerUrls.first, let nftURL = URL(string: "\(explorerURL)/address/\(nftDetailStore.nft.contractAddress)") {
-                      openWalletURL(nftURL)
-                    }
-                  }
-                }) {
-                  HStack {
-                    Text("\(nftDetailStore.nft.contractAddress.truncatedAddress)")
-                    Image(systemName: "arrow.up.forward.square")
-                  }
-                  .foregroundColor(Color(.braveBlurple))
-                }
+                Text(nftDetailStore.nft.contractAddress.truncatedAddress)
+                Image(systemName: "arrow.up.forward.square")
               }
+              .font(.subheadline)
+              .foregroundColor(Color(.braveBlurpleTint))
             }
           }
-          .foregroundColor(Color(.braveLabel))
+          NFTDetailRow(title: Strings.Wallet.nftDetailBlockchain) {
+            Text(nftDetailStore.networkInfo.chainName)
+              .font(.subheadline)
+              .foregroundColor(Color(.braveLabel))
+          }
+          NFTDetailRow(title: Strings.Wallet.nftDetailTokenStandard) {
+            Text(nftDetailStore.nft.isErc721 ? Strings.Wallet.nftDetailERC721 : Strings.Wallet.nftDetailSPL)
+              .font(.subheadline)
+              .foregroundColor(Color(.braveLabel))
+          }
         }
-        if isSVGImage {
-          Text(Strings.Wallet.nftDetailSVGImageDisclaimer)
-            .multilineTextAlignment(.center)
-            .font(.footnote)
-            .foregroundColor(Color(.secondaryBraveLabel))
-            .frame(maxWidth: .infinity)
+        .listRowBackground(Color(.secondaryBraveGroupedBackground))
+      } header: {
+        Text(Strings.Wallet.nftDetailOverview)
+          .listRowInsets(.zero)
+      }
+      if let nftMetadata = nftDetailStore.nftMetadata, let description = nftMetadata.description, !description.isEmpty {
+        Section {
+          Text(description)
+            .font(.subheadline)
+            .foregroundColor(Color(.braveLabel))
+            .listRowBackground(Color(.secondaryBraveGroupedBackground))
```

Comment:
Seems to be something off with the padding on iPad here, but iPhone seems okay 🤔
![iPad](https://github.com/brave/brave-ios/assets/5314553/d8838cfe-4068-4dfc-87df-a713600db42c) | ![iPhone](https://github.com/brave/brave-ios/assets/5314553/90a0ac0f-3c64-4bbb-94be-6cc92384f2df)
--|--

Reply:
seems like its only happens on iOS15 both iphone and ipad.

Reply:
It's not only on iOS 15
<img width="744" alt="Screenshot 2023-11-16 at 10 04 12 AM" src="https://github.com/brave/brave-ios/assets/5314553/191cf335-3ba8-4bd1-97c6-0a70834bc5fa">

Reply:
i was not able to reproduce every time. but when i was, it's always iOS15 only. 
so i addd the row insect for iPad only since you are able to reproduce on iOS17. Please try again, hope it fixes